chapter_linear-regression/weight-decay.md

```diff
@@ -413,6 +442,18 @@ class WeightDecay(d2l.LinearRegression):
         return super().loss(y_hat, y) + self.net.losses
 ```
 
+```{.python .input}
+%%tab jax
+class WeightDecay(d2l.LinearRegression):
+    wd: int = 0
+    
+    def configure_optimizers(self):
+        # weight_decay is not available within `optax.sgd`, but
```

Comment:
What is "weight_decay"? A variable name or a method name?

Reply:
Ahh, it is no name, I'll rephrase this.